sci-electronics/xcircuit/xcircuit-3.9.72.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Circuit drawing and schematic capture program"
+SRC_URI="http://opencircuitdesign.com/xcircuit/archive/${P}.tgz"
+HOMEPAGE="http://opencircuitdesign.com/xcircuit"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="tcl"
+
+DEPEND="
+	app-text/ghostscript-gpl:0=
+	media-libs/fontconfig:1.0=
+	sys-libs/zlib:0=
+	tcl? (
+		dev-lang/tcl:0=
+		dev-lang/tk:0=
+		)
+	x11-libs/cairo:0=
+	x11-libs/libICE:0=
+	x11-libs/libSM:0=
+	x11-libs/libX11:0=
+	x11-libs/libXt:0=
+	x11-libs/libXpm:0=
+"
+RDEPEND=${DEPEND}
+
+src_prepare() {
+	default
+	mv configure.{in,ac} || die
+	eautoreconf "${myeconfargs[@]}"
+}
+
+src_configure() {
+	econf \
+	   $( use_with tcl ) \
```

Comment:
Please fix the messy whitespace and indentation situation here. Tabs for indendation, indent once for each line below econf...
This piece of code was fine before, you would not have needed to drop the myeconfargs var after I told you how to use it.